src/transformers/models/greaselm/processing_greaselm.py

```diff
@@ -0,0 +1,267 @@
+        feature_extractor ([`GreaseLMFeatureExtractor`]):
+            The feature extractor is a required input.
+        tokenizer ([`RobertaTokenizerFast`]):
+            The tokenizer is a required input.
+    """
+    feature_extractor_class = "GreaseLMFeatureExtractor"
+    tokenizer_class = ("RobertaTokenizer", "RobertaTokenizerFast")
+
+    def __init__(self, feature_extractor, tokenizer, max_seq_length=128):
+        super().__init__(feature_extractor, tokenizer)
+        requires_backends(self, ["spacy", "scatter", "sparse"])
+        self.max_seq_length = max_seq_length
+        self.converters = {
+            "commonsenseqa": convert_commonsenseqa_to_entailment,
+            "openbookqa": convert_openbookqa_to_entailment,
+        }
+        feature_extractor.start()
+
+    def __call__(self, question_answer_example: Union[Dict[str, Any], List[Dict[str, Any]]], **kwargs):
+        """
+
+        Main method that takes question-answer example(s) and encodes them into a batch of language model encodings and
+        a batch of graph encodings combining the two encodings ready for GreaseLM model input
+
+        Args:
+        question_answer_example ('Union[Dict[str, Any], List[Dict[str, Any]]]'):
+            The input question answer example. It can be a single example or a list of examples See CommonSenseQA
+            and/or OpenBookQA examples for more information
+
+        Returns:
+              [`dict`]: A [`dict`] with the following fields:
+
+              - input_ids: (batch_size, num_choices, seq_len)
+              - token_type_ids: (batch_size, num_choices, seq_len)
+              - attention_mask: (batch_size, num_choices, seq_len)
+              - special_tokens_mask: (batch_size, num_choices, seq_len)
+              - labels: (batch_size,)
+              - concept_ids: (batch_size, num_choices, max_node_num)
+              - node_type_ids: (batch_size, num_choices, max_node_num)
+              - node_scores: (batch_size, num_choices, max_node_num, 1)
+              - adj_lengths: (batch_size, num_choices)
+              - special_nodes_mask: (batch_size, num_choices, max_node_num)
+              - edge_index: list of size (batch_size, num_choices), where each entry is tensor[2, E]
+              - edge_type: list of size (batch_size, num_choices), where each entry is tensor[E, ]
+        """
+        # Check for valid input
+        if isinstance(question_answer_example, list):
+            assert all([isinstance(e, dict) for e in question_answer_example])
+        elif isinstance(question_answer_example, dict):
+            # add batch dimension
+            question_answer_example = [question_answer_example]
+        else:
+            raise ValueError(
+                "Input parameter 'question_answer_example' must be a "
+                f"Union[Dict[str, Any], List[Dict[str, Any]]] not {type(question_answer_example)}"
+            )
+
+        converter = kwargs.get("question_answer_converter", None)
```

Comment:
Can we add this to the call method of the processor? It's not documented that there's a "question_answer_converter" argument.